src/miniq.js

```diff
@@ -0,0 +1,15 @@
+(function () {
+  'use strict';
+
+  Promise.delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms))
+
+  Promise.defer = () => {
```

Comment:
Both .delay and .defer should be initialized by defineProperty rather than assignment.